### PR TITLE
[MIG] project_stage_state: Migration to 11.0

### PR DESCRIPTION
**Pull request by rodrigets**

### CODE REVIEW

**Review by mstuttgart — APPROVED** [review]

**Comment by rodrigets** [comment]
Good afternoon, @pedrobaeza.
Is there any correction to be made?

**Comment by pedrobaeza** [comment]
@rodrigets module should be reviewed by at least 2 persons and 1 from the PSC. You can ask for reviews in exchange reviewing other PRs for speeding up the merge.

**Comment by rodrigets** [comment]
:+1:

File: project_stage_state/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (elicoidal) [inline project_stage_state/README.rst]: png

File: project_stage_state/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Daniel Reis, 2014
```
Comment (elicoidal) [inline project_stage_state/__init__.py]: no copyright headers in `__init__.py`

File: project_stage_state/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Daniel Reis, 2014
```
Comment (elicoidal) [inline project_stage_state/__manifest__.py]: Copyright 2014 Daniel Reis

File: project_stage_state/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+# GNU Affero General Public License <http://www.gnu.org/licenses/>
+
+{
+    'name': 'Add State field to Project Stages',
+    'version': '11.0.1.0.0',
+    'category': 'Project Management',
+    'summary': 'Restore State attribute removed from Project Stages in 8.0',
+    'author': "Daniel Reis,Odoo Community Association (OCA)",
```
Comment (elicoidal) [inline project_stage_state/__manifest__.py:8]: space after the ','

File: project_stage_state/security/ir.model.access.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_project_task_type_empl,project.task.type.employees,project.model_project_task_type,base.group_user,1,1,0,0
```
Comment (elicoidal) [inline project_stage_state/security/ir.model.access.csv:2]: Add Project manager 1,1,1,1?

**Review by elicoidal — CHANGES_REQUESTED** [review]
Some details.
LGTM but I think the module is thin by itself just defining the state.
IMO it should add onchange relatd function (non blocking though)

**Comment by rodrigets** [comment]
Hi, @elicoidal .
I made the corrections.

**Review by elicoidal — APPROVED** [review]
LGTM

**Pull request merged**